src/main/scala-2.11/org/apache/james/gatling/control/JamesWebAdministrationQuery.scala

```diff
@@ -0,0 +1,17 @@
+package org.apache.james.gatling.control
+
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
+import io.gatling.http.request.builder.HttpRequestBuilder
+
+import java.net.URL
+
+class JamesWebAdministrationQuery(val baseUrl: URL) {
```

Comment:
I find that naming strange. Aren't methods in JamesWebAdministration queries too ?

Reply:
Yes, but they are unlogged/untracked queries, I asked myself the question.